content/v2.0/influx-templates/create.md

```diff
@@ -1,21 +1,105 @@
 ---
-title: Create an Influx Template
+title: Create an Influx template
 description: >
-  placeholder
+  Use the InfluxDB UI and the `influx pkg export` command to create Influx templates.
 menu:
   v2_0:
     parent: Influx templates
     name: Create a template
-    identifier: Create an Influx Template
+    identifier: Create an Influx template
 weight: 101
 v2.0/tags: [influx templates]
 ---
 
-## Placeholder outline
-- Create a template
-  - `influx pkg`
-  - Package everything vs package specific things
-  - Include secrets (only keys are included, uses will still need to populate the secret)
-  - Submit a template to the InfluxDB community template repo
-  - View your package summary
-  - Validate a package
+Use the InfluxDB user interface (UI) and the `influx pkg export` command to
+create Influx templates.
+Add buckets, Telegraf configurations, tasks, and more in the InfluxDB UI and export
+them as a template.
+
+{{% note %}}
+The simplest way to create a template is to create a new organization and build
+the template within the organization.
+**InfluxDB OSS** supports multiple organizations.
+In **InfluxDB Cloud**, your user account is an organization.
+**We recommend using InfluxDB OSS to create Influx templates.**
+{{% /note %}}
+
+**To create a template:**
+
+1. [Start InfluxDB](/v2.0/get-started/).
+2. [Create a new organization](/v2.0/organizations/create-org/).
+3. Use the InfluxDB UI to add resources to your template.
+   The following resources are templatable:
+
+   - [buckets](/v2.0/organizations/buckets/create-bucket/)
+   - [checks](/v2.0/monitor-alert/checks/create/)
+   - [dashboards](/v2.0/visualize-data/dashboards/create-dashboard/)
+   - [dashboard variables](/v2.0/visualize-data/variables/create-variable/)
+   - [labels](/v2.0/visualize-data/labels/)
+   - [notification endpoints](/v2.0/monitor-alert/notification-endpoints/create/)
+   - [notification rules](/v2.0/monitor-alert/notification-rules/create/)
+   - [tasks](/v2.0/process-data/manage-tasks/create-task/)
+   - [Telegraf configurations](/v2.0/write-data/use-telegraf/)
+
+4. Export the template _(see [below](#export-a-template))_.
+
+## Export a template
+There are two methods for exporting a template.
+
+1. Export all resources in an organization _(recommended)_
+2. Export specific resources in an organization
+
+### Export all resources
+Use the `influx pkg export all` command to export all templatable resources
+into a new template manifest.
+Provide the following:
+
+- **Organization name** or **ID**
+- **Authentication token** with read access to the organization
+- **Destination file** path for the template manifest.
+  The extension of the destination file determines the output format of the manifest file.
+  Export template manifests in either YAML (`.yml` or `.yaml`) or JSON (`.json`).
+
```

Comment:
The filename extension determines the template format--both YAML and JSON are supported.